tasks/app2-1.js

```diff
@@ -1,9 +1,22 @@
 import express from 'express';
 import * as userRouter from './routes/users.js';
+import { logger, errorLogger } from './utils/logger.js';
 
 const app = express();
-
 app.listen(3000);
 app.use(express.json());
 
+app.use(logger);
 app.use('/users/', userRouter.router);
+app.use(errorLogger);
+
+app.use((err, req, res, next) => // eslint-disable-line no-unused-vars
+    res.status(500).send(`${err} has occurred!`)
+);
+
+process.on('uncaughtException', (err, req, res, next) => { // eslint-disable-line no-unused-vars
```

Comment:
Consider not passing the unused arguments in the first place to avoid linting issues.
Moreover, consult the dos for the best practices on the matter:
https://nodejs.org/api/process.html#process_event_uncaughtexception